ui/debug.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum, auto
+
+
+class DebugType(Enum):
+    Base = auto()
+    Game = auto()
+    Agent = auto()
+
+
+class Debug:
+    # Turn to false if you want to turn off logging for something
+    ACTIVE_DEBUG_TYPES = {
+        DebugType.Base: True,
+        DebugType.Game: True,
+        DebugType.Agent: True,
+    }
```

Comment:
Added some logging that I want to leave in the program. To turn it off change the value for the category to `False`.